include/rocksdb/utilities/customizable_util.h

```diff
@@ -15,6 +15,13 @@
 #include "rocksdb/utilities/object_registry.h"
 
 namespace ROCKSDB_NAMESPACE {
+// The FactoryFunc functions are used to create a new customizable object
```

Comment:
For this file, can we create a customizable_util.cc in the /util and here, we only keep the declarations here？